examples/gno.land/r/demo/markdown_test/markdown.gno

```diff
@@ -143,6 +143,64 @@ but that looks ugly.
       You just need to indent an additional four spaces.
 
 
+Columns
+=======
+
+Columns can be created using the ` + "`<gno-columns>`" + ` tag. The columns will be defined based on headings, with the first heading setting the level reference for the separator.
+
+Example using ` + "`<gno-columns>`" + `:
+
+<gno-columns>
+## Column 1
+
+content 1
+
+## Column 2
+
+content 2
+</gno-columns>
+
+You can also use ` + "`:::`" + ` as a shortcut for creating columns:
```

Comment:
let's start with just `<gno-columns>`. 

`:::` seems too 'precious' to be an alternative to something that's already concise and clear enough.
if ever we really want it, we can add it in another PR.